src/Gui/ManualAlignment.cpp

```diff
@@ -857,7 +857,7 @@ void ManualAlignment::continueAlignment()
         grp.addToViewer(myViewer->getViewer(0));
         grp.setAlignable(true);
 
-        Gui::getMainWindow()->showMessage(tr("Please pick points in the left and right view"));
+        Gui::getMainWindow()->showMessage(tr("Selected points in the left and right view"));
```

Comment:
```suggestion
        Gui::getMainWindow()->showMessage(tr("Select points in the left and right view"));
```
Fixing a typo in my previous suggestion.